sycl/test-e2e/ClangOffloadWrapper/no-sycl-rdc.cpp

```diff
@@ -0,0 +1,63 @@
+// Test clang-offload-wrapper with -fno-sycl-rdc
+
+// REQUIRES: gpu && level_zero
+// UNSUPPORTED: cuda, hip
```

Comment:
This line isn't needed after the previous. What's the reason of the limitation? Is that just due to missing `-fsycl-targets` in the compilation command?

Reply:
Let me try to remove it and see what happens, thanks

Reply:
nvm we need l0, we dont support cuda/hip for the new device linking backend yet